aptos-move/parallel-executor/src/scheduler.rs

```diff
@@ -81,6 +81,7 @@ pub enum SchedulerTask<'a> {
 ///    ↓                finish_abort
```

Comment:
line 74 in particular, can you update the comment please?

Reply:
good catch!